partcad/src/partcad/sketch_factory_dxf.py

```
@@ -7,24 +7,40 @@
+sys.path.append(os.path.join(os.path.dirname(__file__), "wrappers"))
+from ocp_serialize import register as register_ocp_helper
 
-class SketchFactoryDxf(SketchFactoryFile):
+
+class SketchFactoryDxf(SketchFactoryPython):
     tolerance = 0.000001
     include = []
     exclude = []
 
-    def __init__(self, ctx, source_project, target_project, config):
+    def __init__(self, ctx, source_project, target_project, config, can_create=False):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)